### ISSUE TITLE
Improve CLI

### ISSUE THREAD

**Issue by ayan-b** [yunhailuo/xena-GDC-ETL#16]
Currently, the user requires to type `python file_name args` in order to perform CLI operations. But, when the project will be installed using `setup.py`, the files will reside at `env/site_packages`, and thus will be hard for the end-user.

It can be improved by using a single CLI command, for example, `xena-gdc-etl gdc2xena args`
and `xena-gdc-etl gdc_check_new args`. In this way, the user needs not worry about path to the python files.

See also: https://github.com/yunhailuo/xena-GDC-ETL/pull/17#issuecomment-475110147

**Comment by ayan-b** [comment]
@yunhailuo  Would like work on this once #13 is merged.

**Comment by yunhailuo** [comment]
Sounds like a good feature to have. I just merged #13.

Two potential suggestions:

1. Feel free to prioritize scripts and we don't have to get all scripts in in one commit/PR. Actually better have a framework first then figure out specifics
2. Some clean up is going to be helpful. For example, I found `make_metadata.py` overlaps with `xena_dataset` module quite a lot. Such duplication is bad for maintenance. Also, some scripts may not be useful or relevant anymore.
